spec/models/manageiq/providers/vmware/infra_manager/metric_capture_spec.rb

```diff
@@ -1,4 +1,4 @@
-require Rails.root.join('spec/tools/vim_data/vim_data_test_helper')
+require File.join(ManageIQ::Providers::Vmware::Engine.root, 'spec/tools/vim_data/vim_data_test_helper.rb')
```

Comment:
You can also 
```
require ManageIQ::Providers::Vmware::Engine.root.join('spec/tools/vim_data/vim_data_test_helper.rb').to_s
```
or, what I do in aws `require_relative` and keep the file close to where it's actually used.
Like `vim_data_test_helper` seems to be only used by specs in `vmware/infra_manager` - why not put it here?